src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2Serializer.cs

```diff
@@ -292,7 +295,10 @@ public virtual Saml2Assertion ReadAssertion(XmlReader reader)
                 }
 
                 // attach signedXml for validation of signature
```

Comment:
nit: seem like one this comment doesn't belong.

Reply:
removed, thanks.